FreeRTOS/FreeRTOS
# Build: Fix Demo/RX100_RX113-RSK_Renesas_e2studio RTOS demo project build error

Description
-----------
This PR fixes Demo/RX100_RX113-RSK_Renesas_e2studio RTOS demo project build error.
There are no changes in the runtime functionality.

Test Steps
-----------
I did the following steps on the e2 studio v7.8.0 and I used CC-RX V3.02.

(1) Import the project
(2) Update legacy e2 studio projects
(3) Select toolchain type and toolchain version
(4) Clean all projects and start a build

But I don't have the evaluation board therefore I can't check the program which are built.
If this isn't acceptable for you, please let me know.

Related Issue
-----------
Update Renesas GCC compiler ports #135
https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/135#issuecomment-680172086
https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/135#issuecomment-679215454

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
